packages/client-react-streaming/src/ReadableStreamLink.tsx

```diff
@@ -56,6 +56,13 @@ export const TeeToReadableStreamLink = new ApolloLink((operation, forward) => {
   const controller = context[teeToReadableStreamKey];
 
   if (controller) {
+    const tryClose = () => {
+      try {
+        controller.close();
+      } catch {
+        // maybe we already tried to close the stream, nothing to worry about
+      }
+    };
```

Comment:
This can be triggered from multiple different places, and we want the first call to close and the later calls to not error.